src/pages/settings/Preferences/PriorityModesPage.js

```diff
@@ -0,0 +1,99 @@
+import _, {compose} from 'underscore';
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+
+import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import CONST from '../../../CONST';
+import OptionsList from '../../../components/OptionsList';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+import themeColors from '../../../styles/themes/default';
+import * as Expensicons from '../../../components/Icon/Expensicons';
+import ONYXKEYS from '../../../ONYXKEYS';
+import * as User from '../../../libs/actions/User';
+
+const greenCheckmark = {src: Expensicons.Checkmark, color: themeColors.success};
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const PriorityModesPage = (props) => {
+    const priorityModes = _.map(
+        [
+            {
+                value: CONST.PRIORITY_MODE.DEFAULT,
+                text: props.translate('priorityModesPage.mostRecent'),
+                alternateText: props.translate('priorityModesPage.mostRecentModeDescription'),
+            },
+            {
+                value: CONST.PRIORITY_MODE.GSD,
+                text: props.translate('priorityModesPage.focus'),
+                alternateText: props.translate('priorityModesPage.focusModeDescription'),
+            },
+        ], mode => (
+            {
+                ...mode,
+                keyForList: mode.value,
+
+                // Include the green checkmark icon to indicate the currently selected value
+                customIcon: props.priorityMode === mode.value ? greenCheckmark : undefined,
+
+                // This property will make the currently selected value have bold text
+                boldStyle: props.priorityMode === mode.value,
+            }
+        ),
```

Comment:
I wonder if it's time to try to extract this into a function? Since we do something similar for pronouns & probably timezones?

Reply:
oh, good catch, I'll consider this in the follow-up PR which is going to align how list selection looks in the App. [Here's](https://github.com/Expensify/App/issues/11795#issuecomment-1408817926) the issue for it

Reply:
Do you agree with it? Can I resolve this convo?

Reply:
The refactor would be pretty small and this isn't a critical to merge immediately so i would say lets do this here?

Reply:
As we discussed in our 1:1, we better discuss this in the follow-up issue about option and selection lists to avoid adding many out of scope changes in this PR